relay-metrics/src/aggregation.rs

```diff
@@ -758,6 +752,9 @@ enum AggregateMetricsErrorKind {
     /// A metric bucket had invalid characters in the metric name.
     #[fail(display = "found invalid characters")]
     InvalidCharacters,
+    /// A metric bucket had an unknown namespace in the metric name.
+    #[fail(display = "found invalid namespace")]
```

Comment:
_nit_: The namespace variant is called `Unsupported` rather than "invalid", we could align that term. I do not have a preference.